pkg/api-server/server.go

```diff
@@ -135,6 +138,10 @@ func NewApiServer(resManager manager.ResourceManager, defs []definitions.Resourc
 	} else {
 		container.ServeMux.HandleFunc("/gui/", newApiServer.notAvailableHandler)
 	}
+
+	// Install the custom WS
```

Comment:
nit: I don't think this comment is needed. I like commenting out code for the following reasons
1) explaining _why_ something is happening
2) explaining code design and design decisions / tradeoffs
3) explaining difficult code when language is crippling the developer to write an easy to understand code

Here it's pretty obvious what is going on